analyzers/rolaguard_base_analyzer/BaseAnalyzerMain.py

```diff
@@ -273,13 +279,54 @@ def process_packet(packet, policy):
         device.max_lsnr < policy.get_parameters("LAF-102")["minimum_lsnr"]
     ):
         emit_alert(
-            "LAF-102", 
+            alert_type="LAF-102", 
             packet=packet,
             device=device,
             device_session=device_session,
             gateway=gateway,
             lsnr=device.max_lsnr
         )
 
+    if packet.is_retransmission:
```

Comment:
I think it is better if we put all the code related to LAF-103 in another file, and follow the same interface of ABPDetector or CheckSessionRegeneration. Therefore, In the future we can use them as a independant plugins.

Reply:
I will extract the code to a new class